compass_app/app/lib/data/repositories/itinerary_config/itinerary_config_repository_memory.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:async';
 
 import 'package:compass_model/model.dart';
+import 'package:flutter/material.dart';
```

Comment:
nit: same, I don't think this is used.

Reply:
Sorry, I forgot to do one last cleanup before opening the PR. All the unused imports are removed now.